src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/client/PrisonRegisterClient.kt

```diff
@@ -44,7 +44,7 @@ class PrisonRegisterClient(
   }
 
   fun getPrisonContactDetails(prisonCode: String): Optional<PrisonRegisterContactDetailsDto> {
-    val uri = "/secure/prisons/id/$prisonCode/contact-details?departmentType=prison"
+    val uri = "/secure/prisons/id/$prisonCode/department/contact-details?departmentType=social_visit"
```

Comment:
Too late as it's merged, but I don't think this will work. It needs to be `SOCIAL_VISIT` (case sensitive)